src/modules/request-recorder.js

```diff
@@ -102,7 +102,7 @@ module.exports = (opts) => {
             });
             await new Promise((resolve) => {
               options.protocol = `${protocol}:`;
-              const r = { http, https }[protocol].request(options, (response) => {
+              const r = { http, https: http }[protocol].request(options, (response) => {
```

Comment:
Why are we doing this?

Reply:
haha good question

Reply:
it's so that we can support other protocols easily in the future